src/Records/CAA.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Spatie\Dns\Records;
+
+/**
+ * @method int flags()
+ * @method string tag()
+ * @method string value()
+ */
+class CAA extends Record
+{
+    protected int $flags;
+    protected string $tag;
+    protected string $value;
+
+    public static function parse(string $line): self
+    {
+        $attributes = static::lineToArray($line, 7);
+
+        return static::make([
+            'host' => $attributes[0],
+            'ttl' => $attributes[1],
+            'class' => $attributes[2],
+            'type' => $attributes[3],
+            'flags' => $attributes[4],
+            'tag' => $attributes[5],
+            'value' => $attributes[6],
+        ]);
+    }
+
+    public function __toString(): string
+    {
+        return sprintf(
```

Comment:
I think using regular string concatenation would be more readable.

Reply:
I think regular concatenation not but string interpolation could be helpful.